microsoft-edge/progressive-web-apps/how-to/index.md

```diff
@@ -1,184 +1,197 @@
-        "description": "A basic temperature converter application that can convert to and from Celsius, Kelvin, and Fahrenheit",
-        "start_url": "/",
-        "background_color": "#2f3d58",
-        "theme_color": "#2f3d58",
-        "orientation": "any",
-        "display": "standalone",
-        "icons": [
-            {
-                "src": "/icon512.png",
-                "sizes": "512x512"
-            }
-        ]
-    }
-    ```
 
-1.  Add a 512x512 pixel app icon image named `icon512.png` to your project. You can use the [sample image](./index-images/icon512.png) for testing purposes.
+<!-- ====================================================================== -->
+## Back-end code, files, endpoints, and data (server-side code)
 
-1.  In Visual Studio Code, open `index.html`, and add the following code inside the `<head>` tag.
+A PWA potentially has back-end code that resides and runs on the web server.
 
-    ```html
-    <link rel="manifest" href="/manifest.json">
-    ```
+The temperature converter sample Progressive Web App (PWA) doesn't have any server-side code, because the app exclusively runs on the device it's installed on, and doesn't need any server-side data.
 
-   The above code snippet links the new web app manifest file to your website.
+Like a web app, a Progressive Web App (PWA) can include some _back-end code_ (_server-side code_) that implements any web service API endpoints that are needed by your app, when connected to the internet, to retrieve dynamic content that may be stored in a database on your server.
 
-Your VS Code project should now look somewhat like this:
+The back-end code for a Progressive Web App (PWA) can use the server-side languages of your choice, such as:
+* ASP.NET
+* Java
+* Node.js
+* PHP
 
-![VS Code showing the sample PWA project, with the index.html, manifest.json, and icon files](./index-images/visual-studio-project-with-manifest.png)
+Server-side web service API endpoints might not be required, depending on the app your're building.
+
+The remaining sections explain the files that make up the PWA sample.
 
 
 <!-- ====================================================================== -->
-## Step 4 - Continue building the user interface of your app
+## The app's initial webpage (`index.html`)
+
+`index.html` defines the first page that users see when they access the Progressive Web App (PWA).
+
+[index.html](https://github.com/MicrosoftEdge/Demos/blob/main/pwa-getting-started/index.html) defines the webpage layout of the Progressive Web App (PWA).  The HTML describes the content in the app, including the text, images, text fields, and buttons that appear in the user interface.
+
+```html
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="icon" type="image/png" href="https://edgestatic.azureedge.net/welcome/static/favicon.png">
```

Comment:
bug in Demos repo /Demos/pwa-getting-started/
the temperature convertor sample code uses `/static/favicon.png` 
but should instead use `/pwa-getting-started/icon512.png`
@captainbrosset 

live pwa: 
https://microsoftedge.github.io/Demos/pwa-getting-started/